website/core/Footer.js

```diff
@@ -8,21 +8,51 @@ const PropTypes = require('prop-types');
 
 const React = require('react');
 
-const GitHubButton = props => (
-  <a
-    className="github-button" // part of the https://buttons.github.io/buttons.js script in siteConfig.js
-    href={`https://github.com/${props.config.organizationName}/${props.config.projectName}`}
-    data-icon="octicon-star"
-    data-count-href={`/${props.config.organizationName}/${props.config.projectName}/stargazers`}
-    data-show-count="true"
-    data-count-aria-label="# stargazers on GitHub"
-    aria-label="Star this project on GitHub"
-  >
-    Star
-  </a>
+const SocialFooter = props => (
+  <div>
+    <div>
+      <h5>GitHub</h5>
```

Comment:
I think we don't need the headings. The social buttons are pretty self-describing in my opinion. I think it looks cleaner without them.

What do you think?

Reply:
Simulated iphone X

<img width="191" alt="iphone x" src="https://user-images.githubusercontent.com/17883920/40189571-5f297584-5a2f-11e8-85bb-9e50790155b6.PNG">

Reply:
That looks good to me. #2, then?

Reply:
PR commit updated using #2.
Extra thing to note is that the share dialog button won't work due to the app not being set to public. Issue related is #625 .

I think we also need to update the doc on `facebookAppId` so that user will set their app to public (not development mode)